Sources/PaymentButtons/PayPalPayLaterButton.swift

```diff
@@ -27,7 +27,7 @@ public final class PayPalPayLaterButton: PaymentButton {
         insets: NSDirectionalEdgeInsets? = nil,
         color: Color = .gold,
         edges: PaymentButtonEdges = .softEdges,
-        size: PaymentButtonSize = .collapsed,
+        size: PaymentButtonSize = .regular,
```

Comment:
Same here on docstrings updates

Reply:
Looks like the docstring on line 25 was still missed